src/shared/data/Tile-list.es6

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+//
+// Tile-lis
+// Created by dpekar on 10/7/15.
+//
+
+module.exports = {
+
+    'basic': [
+        {
+          name: 'Suburbs',
+          cost: 3,
+          category: 'Residential',
```

Comment:
When using string constants like this, we should define them in a class or module.  e.g.:

```
const TileCategories = require('...')
...
  {
    name: 'Suburbs',
    cost: 3,
    category: TileCategories.RESIDENTIAL,
    icon: TileIcons.SCHOOL,
    stage: GameStages.BASIC,
    ...
}
```